src/Utility.hpp

```diff
@@ -131,6 +131,20 @@ void logv(int levelFlags, String input, ...);
 
 #endif // DEBUG_LEVEL>0
 
+// For some reason arduino just defines all float functions to be as double
```

Comment:
did you test this also on esp32? on AVR float and double are actually the same (32bit). but what if it is different for esp32? could we break some external libs this way?

Reply:
I haven't tested on esp32 (I don't have the hardware) but this really shouldn't break anything. Worst case scenario is that a `double` is implicitly demoted to `float` but that can be fixed with adding `-Wconversion` in a later PR (decent amount of code needs to be fixed in order to turn that one on haha)

Reply:
Yeah good idea. would be a bummer if some esp32 library really needs double precision and we break it this way.